build/AzurePipelinesTemplates/ProjectReunion-RunHelixTests-Job.yml

```diff
@@ -14,15 +14,15 @@ parameters:
   matrix: 
     Release_x86:
       buildPlatform: 'x86'
-      buildConfiguration: 'release'
+      buildConfiguration: 'Release'
       openHelixTargetQueues: 'windows.10.amd64.client19h1.open.xaml%3bwindows.10.amd64.clientrs3.open.xaml'
-      outputDirectory: BuildOutput\Release\x86\ProjectReunionHelix\
+      outputDirectory: BuildOutput\Release\x86\BaseUnitTests\
```

Comment:
>BaseUnitTests [](start = 47, length = 13)

if we need to do this for every unit tests, it's lots of extra work.

Reply:
Not quite sure what you mean here.  For every unit test we will need to build both flavors and run them on the helix queue.